cmd/state/internal/cmdtree/artifacts.go

```diff
@@ -71,6 +76,11 @@ func newArtifactsDownloadCommand(prime *primer.Values) *captain.Command {
 				Description: locale.Tl("artifacts_download_flags_commit_description", "The commit ID to download artifacts from"),
 				Value:       &params.CommitID,
 			},
+			{
+				Name:        "target",
+				Description: locale.Tl("artifacts_flags_target_description", "The target to download artifacts from"),
```

Comment:
This is a duplicate locale key.

Reply:
It's only duplicated between artifacts and artifacts dl, which are so closely related that I would consider this a "feature".

Reply:
I meant it's duplicated with another key added to this PR: https://github.com/ActiveState/cli/pull/3234/files#diff-ba71f664d43f1a5f805cd749ceab31582d01498132c3fdb97fc9bc941523058bR38

If this is intentional, then please add to en-us.yaml so that shared locale keys always use the same value.